gem/lib/pupperware/spec_helper.rb

```diff
@@ -101,7 +102,7 @@ def docker_compose(command_and_args)
     file_arg = File.file?(overrides) ? "--file #{overrides}" : ''
     run_command("docker-compose --file docker-compose.yml #{file_arg} \
                                 --no-ansi \
-                                #{command_and_args}")
+                                #{command_and_args}", stream: StringIO.new)
```

Comment:
How does this actually do anything?

Since we're not saving the `StringIO` object in a variable and referencing it after this call...

Reply:
Intended to eat the output / silence redundant information... since `inspect_container` already emits the container id / query and output, there's no reason to also emit the same container id multiple times from `run_command` as well.